routes/api.js

```diff
@@ -0,0 +1,7 @@
+const { Router } = require('express');
+const adsRouter = require('./api/ads');
+const router = Router();
+
+router.use('/ads', adsRouter);
```

Comment:
Lamento que te enteraras tan tarde u-u, pero desde los embajadores de BDD2 cambiamos el nombre `ads` por que provocaba problemas con ad-blocks, de momento el unico nombre que quedo fue `pop-ups` creo, cualquier cosa me corrijen, pero yo le pondria ese

FYI @Paulaforero @HopeAero @Edusus @Nannajpg @Noralgorithm 
```suggestion
router.use('/pop-ups', adsRouter);
```